### DISCUSSION TITLE
Question: Does this hide the menu bar and take over in its place?

### DISCUSSION THREAD

**Discussion by iCraft85** [FelixKratz/SketchyBar#225] [Q&A] [answered]
So I wanted to know if this replaces the menubar so that it wont show, cuz I dont rly want the menu bar to be shown also at the same time, also if it does how does it handle the notch that is present on 14" and 16" macs?

**Comment by IoannisTM** [comment, marked as answer]
the menubar you can set it to autohide.
So you will always have visible the Sketchybar, and menubar will only appear if you move your mouse to the top.

for the notch, there is a split bar config , and there is also an entry to set the width of the notch so it will not be able to utilized by items.
you can check the property here https://felixkratz.github.io/SketchyBar/config/bar
